raiden/utils/rsk_gas_price_strategy/rsk_gas_price_strategy.py

```diff
@@ -172,7 +172,7 @@ def time_based_gas_price_strategy(web3, _transaction_params):
         else:
             latest_block_gas_price = int(latest_block_gas_price)
         gas_price = _compute_gas_price(probabilities, probability / 100)
-        max_gas = max(gas_price, latest_block_gas_price)
+        max_gas = max(gas_price, latest_block_gas_price * 1.10)
```

Comment:
lets add:

- docstring that explains why 1.10
- a TODO related with this issue https://jirainfuy.atlassian.net/browse/RLP-992